### DISCUSSION TITLE
Collabora container seems multiplied

### DISCUSSION THREAD

**Discussion by SamuXzX** [nextcloud/all-in-one#1315] [Questions] [answered]
When I enable Nextcloud Office it seems that the process gets multiplied, taking up to 30% of the memory (which is aproximately 4G):
![immagine](https://user-images.githubusercontent.com/30182251/198021877-d5896c05-e190-44e3-a91f-89a10583f203.png)

At the moment of this report I'm not using Nextcloud Office, it simply starts this way.

What can I do? Should I open an Issue?

**Comment by Zoey2936** [comment, marked as answer]
Sorry, but I didn't understand the exact problem.
Is the problem that it takes much RAM or that the container starts even if you disable Collabora in the AIO-Interface?

**Reply by SamuXzX** [reply]
In the list of processes it appears three times in three different networks. Each time eats 10% of RAM. Is it normal? Why am I seeing three processes started from "coolforkit" even if I'm not actually editing a document?

**Reply by Zoey2936** [reply]
Collabora runs as a service, this means there are always processes running in the background. 
For the memory, I think this normal, if you don't think so, you could report this to [Collabora](https://github.com/CollaboraOnline/online)
Since nextcloud-aio useses the official Collabora Online Image: https://github.com/nextcloud/all-in-one/blob/main/Containers/collabora/Dockerfile and added only a health check and the option to set the timezone - this should normally not influence the memory consumption.
For the multiple processes, this is in my opinion normal that a service creates multiple processes.

**Reply by SamuXzX** [reply]
Thank you very much.
